airflow/cli/commands/scheduler_command.py

```diff
@@ -30,6 +32,8 @@
 @cli_utils.action_logging
 def scheduler(args):
     """Starts Airflow Scheduler"""
+    skip_serve_logs = args.skip_serve_logs
```

Comment:
I wonder if this should be the opposite - opt in to log serving. There is probably more usage that wouldn't want this on by default than the other way around (e.g. running locally).

Reply:
This is only ON by default for `LocalExecutor` and `SequentialExecutor` though. And kept this behaviour to replicate what we have on it our Worker (`airflow celery worker`)

Reply:
Ah, consistency with worker makes sense. I don't have a strong opinion here, just when looking at scheduler in isolation the default seems backwards to me.